spec/spec_helper.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+require 'webmock/rspec'
+
+WebMock.disable_net_connect!(allow_localhost: true)
```

Comment:
```suggestion
WebMock.disable_net_connect!(allow_localhost: true, allow: 'chromedriver.storage.googleapis.com')
```
We'll want to add this exception so that `webdrivers` can request chromedriver.

Reply:
👌